packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1920,7 +1924,9 @@ function performWork(minExpirationTime: ExpirationTime, dl: Deadline | null) {
   }
   // If there's work left over, schedule a new callback.
   if (nextFlushedExpirationTime !== NoWork) {
-    scheduleCallbackWithExpirationTime(nextFlushedExpirationTime);
+    scheduleCallbackWithExpirationTime(nextFlushedExpirationTime, {
+      continue: true,
```

Comment:
`continue` should only be true if we were already inside an async callback. That's the check above: `deadline !== null`. Maybe scheduler should warn or error if `continue: true` is passed but you're not already inside a callback.

To me this indicates that the API should use generators instead. Then this type of mistake wouldn't be possible.